src/pages/about/index.tsx

```diff
@@ -14,73 +14,105 @@ const Explanation = () => {
+          goes to each team. At least 50% of teams need to approve a set of
+          scores by majority vote – otherwise, the ETH stays in the pot.
+        </p>
+        <p className="my-4">
+          Everything runs onchain, making Defifa games decentralized,
+          uncensorable, and unstoppable. No third parties needed.
         </p>
       </div>
 
       <h2 className="text-4xl md:text-4xl text-center font-medium mt-12 mb-6">
         Game Stages
       </h2>
       <div className="mx-auto max-w-4xl" style={{ textIndent: 50 }}>
-        <p className="my-4">A Defifa game happens in four stages:</p>
+        <p className="my-4">A Defifa game happens in five stages:</p>
 
         <ol>
           <li className="my-2">
-            <b>1. Create:</b> Anyone can create a Defifa game. Game creators
-            represent different predictions as NFTs, decide the price of each
-            NFT, and set the game's schedule. Game creators also have the option
-            to reserve a percentage of the final pot or a percentage of the NFTs
-            for their team, for NFT artists, or for someone else.
+            <b>1. Create:</b> Anyone can create a Defifa game. Game creators set
+            up teams, set the prices of each team's NFT, and set the game's
+            schedule. Game creators can choose to reserve a percentage of the
+            pot or a percentage of the NFTs for the recipients of their
+            choosing.
           </li>
           <li className="my-2">
-            <b>2. Mint:</b> People make their predictions by minting NFTs – one
-            person can mint multiple NFTs corresponding to multiple predictions.
-            After an amount of time set by the game's creator, NFT minting
-            closes.
+            <b>2. Mint:</b> People can join teams by minting their NFTs. One
+            person can mint multiple NFTs from multiple teams. After an amount
+            of time set by the game's creator, minting closes.
           </li>
           <li className="my-2">
-            <b>3. Score:</b> Once a game finishes, anyone can score the game's
-            results to determine how much of the pot will go to each
-            prediction's NFTs. At least 50% of the NFTs for at least 50% of the
-            predictions have to vote for a set of scores – otherwise, people can
-            reclaim their ETH from the pot.
+            <b>3. Refunds:</b> A refund window opens if the game's creator set
+            one up. During this window, anyone can redeem their NFT to reclaim
+            their ETH from the pot. <b>This stage is optional.</b>
           </li>
           <li className="my-2">
-            <b>4. Earn:</b> If a set of scores is approved, people will be able
-            to burn their NFTs to claim their share of the rewards based on
-            their predictions.
+            <b>4. Score:</b> Once a game ends, anyone can score the game's
```

Comment:
The timing here for submitting a scorecard is variable and set at game deploy (currently not in ux). We are defaulting to as soon as Game Starts (mint and refund end) anybody may submit a score.

Reply:
Fixing